domains/alora.json

```
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "aloramiaa",
+        "email": "xaloramia@gmail.com"
```

nvm just noticed, missing a comma here
```suggestion
        "email": "xaloramia@gmail.com",
```